posts/2021-01-22-ee9-messaging-security-21002-beta.adoc

```diff
@@ -0,0 +1,205 @@
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#fault[MicroProfile Fault Tolerance 3.0]
+* link:https://openliberty.io/blog/2020/09/02/jakarta-microprofile-beta-200010.html#health[MicroProfile Health 3.0]
+* link:https://openliberty.io/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[MicroProfile OpenAPI 2.0]
+* link:https://openliberty.io/blog/2020/11/24/microprofile-rest-client-kubernetes-secrets-21001beta.html#MP[MicroProfile Rest Client 2.0]
+* link:https://openliberty.io/blog/2020/10/27/microprofile40-jakartaee9-beta-200012.html#jwt[MicroProfile JWT 1.2]
+* link:https://openliberty.io/blog/2020/11/24/microprofile-rest-client-kubernetes-secrets-21001beta.html#kube[Using Kubernetes secrets in Liberty variables]
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.2-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.2-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have a number of new API Release Candidates to join the expanding library of supported Jakarta packages.
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* <<messaiging, Messaging 3.0 (`messaging-3.0, messagingClient-3.0, messagingServer-3.0, messagingSecurity-3.0`)>>
+* <<security, Security 2.0 (`appSecurity-4.0, appSecurityClient-1.0`)>>
+* Batch 2.0 (`batch-2.0`)
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+* Jakarta WebSocket 2.0 (`websocket-2.0`; now with full CDI integration)
+* RESTful Web Services 3.0 (`restfulWS-3.0` and `restfulWSClient-3.0`)
+* Jakarta Server Faces 3.0 (`faces-3.0`)
+* Jakarta Connectors 2.0 (`connectors-2.0`)
+* Jakarta Enterprise Beans 4.0 (`enterpriseBeans-4.0`)
+* Jakarta Enterprise Beans Remote 4.0 (`enterpriseBeansRemote-4.0`)
+* Jakarta Enterprise Beans Home 4.0 (`enterpriseBeansHome-4.0`)
+* Jakarta Enterprise Beans Lite 4.0 (`enterpriseBeansLite-4.0`)
```

Comment:
Do we want to include enterpriseBeansPersistentTimer-4.0 feature?  It would look like:  Jakarta Enterprise Beans Persistent Timers 4.0 ('enterpriseBeansPersistentTimer-4.0')

Reply:
Added